code/game/atoms.dm

```diff
@@ -464,4 +464,7 @@ its easier to just keep the beam vertical.
 	if((!message))
 		return
 	for(var/mob/O in hearers(src, null))
-		O.show_message("<span class='game say'><span class='name'>[src]</span> [atom_say_verb], \"[message]\"</span>",2)
\ No newline at end of file
+		O.show_message("<span class='game say'><span class='name'>[src]</span> [atom_say_verb], \"[message]\"</span>",2)
+
+/atom/proc/update_addictions(var/datum/reagent/R, var/amount)

```

Comment:
There was initially a reason, but I believe I moved some things around partway and forgot to adjust this. Will fix this evening